packages/lesswrong/lib/collections/sequences/helpers.ts

```diff
@@ -13,6 +13,13 @@ export const sequenceGetPageUrl = function(sequence, isAbsolute = false){
   return `${prefix}/s/${sequence._id}`;
 };
 
+export const getCollectionOrSequenceUrl = function (sequence, isAbsolute = false) {
```

Comment:
nit: I think the rest of the codebase is just using anonymous functions(`(args) => { //logic }`), rather than unnamed functions (`function (args) { //logic }`)

Reply:
also if we have a type for `sequence` it'd be good to include it in the function signature, as well as `boolean?` for `isAbsolute`